requirements/test-requirements.txt

```diff
@@ -21,7 +21,9 @@ ipython
 # 1.1.0 contains this issue: https://github.com/microsoft/FLAML/issues/871
 flaml!=1.1.0
 # Required by custom flavor example
-sktime==0.16.0
+sktime>=0.16.0
```

Comment:
Needed to bump to run on 3.11, can repin if needed.

Reply:
I think this is fine